src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -7,20 +7,19 @@
 /**
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
  */
 class MembershipApplicationTrackingInfo {
 
-	private $totalImpressionCount;
-	private $singleBannerImpressionCount;
-	private $confirmationPage;
-	private $confirmationPageCampaign;
+	private $campaignCode;
+	private $keyword;
 
-	public function __construct( int $totalImpressionCount, int $singleBannerImpressionCount,
-		string $confirmationPage, string $confirmationPageCampaign ) {
-		$this->totalImpressionCount = $totalImpressionCount;
-		$this->singleBannerImpressionCount = $singleBannerImpressionCount;
-		$this->confirmationPage = $confirmationPage;
-		$this->confirmationPageCampaign = $confirmationPageCampaign;
+	public function __construct( string $campaignCode, string $keyword ) {
+		$this->campaignCode = $campaignCode;
+		$this->keyword = $keyword;
 	}
 
+	public function getInfo() {
+		return implode( '/', array_filter( [ $this->campaignCode, $this->keyword ] ) );
```

Comment:
This class is meant to be a value object. This transformation is either something that belongs into the presentation layer or the data access one, I'm not sure which one.